client/blocks/reader-combined-card/post.jsx

```diff
@@ -0,0 +1,135 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import { has, get } from 'lodash';
+import ReactDom from 'react-dom';
+import closest from 'component-closest';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import AutoDirection from 'components/auto-direction';
+import ReaderExcerpt from 'blocks/reader-excerpt';
+import ReaderVisitLink from 'blocks/reader-visit-link';
+import ReaderAuthorLink from 'blocks/reader-author-link';
+import { recordPermalinkClick } from 'reader/stats';
+import PostTime from 'reader/post-time';
+import ReaderFeaturedImage from 'blocks/reader-featured-image';
+import ReaderFeaturedVideo from 'blocks/reader-featured-video';
+import * as stats from 'reader/stats';
+
+class ReaderCombinedCardPost extends React.Component {
+	static propTypes = {
+		post: React.PropTypes.object.isRequired,
+		streamUrl: React.PropTypes.string,
+		onClick: React.PropTypes.func,
+	};
+
+	propagateCardClick = () => {
```

Comment:
can we ever have discover cards in here?
If not maybe remove the func and put `this.props.onClick( this.props.post ) ` within handleCardClick.